boards/arm/thingy53_nrf5340/Kconfig

```diff
@@ -99,3 +99,19 @@ config DOMAIN_CPUAPP_BOARD
 	  another board.
 
 endif # BOARD_THINGY53_NRF5340_CPUNET
+
+config PMIC_PWM_MODE
+	bool "Set PMIC to PWM mode"
+	default n
+
+if PMIC_PWM_MODE
+
+config PMIC_PWM_MODE_PIN
```

Comment:
Should this be user-assignable on Thingy:53? This is actually constant for Thingy:53 board.